app/controllers/clients/files_controller.rb

```diff
@@ -27,7 +27,7 @@ def index
       end
 
       @consent_files = consent_scope
-      @files = file_scope.page(params[:page].to_i).per(20).order(created_at: :desc)
+      @pagy, @files = pagy(file_scope.order(created_at: :desc))
```

Comment:
```suggestion
      @files = file_scope.order(created_at: :desc)
      @pagy, @files = pagy(@files)
```
It seems elsewhere you applied pagy separately, which seems like it might make searching easier

Reply:
I was having this same thought as I've been reviewing.  It's way easier to follow if we set the variable first.  That said, I'm not sure it's worth the effort, and there's a subtle oddness of re-using the same variable once as a scope and then immediately after as an array.